generators/app/templates/e2e/e2e.js

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
Jie, this test should not be targeting index.html, there should be such an e2e test generated for each action. 
When running the add-action generator, this should generate the e2e test together with the action. Probably the best place to add this logic would be in `lib/ActionGenerator` and in each `generators/add-action/..` generators.
Note that this will be automatically picked up by the `app` generator which will internally run the `add-action` generator.
Also don't forget to clean up any associated e2e test when running the `delete-action`  generator.

when running the `add-action` generator you could create a file `e2e/<action-name>.js` and in the test you can retrieve the URL to call the action like this: 
```javascript
const { Config } = require('@adobe/aio-sdk').Core
const fs = require('fs')
const namespace = Config.get('runtime.namespace') 
const hostname = Config.get('cna.hostname') || 'adobeio-static.net'
const packagejson = JSON.parse(fs.readFileSync('../../package.json').toString())
const runtimePackage = `${packagejson.name}-${packagejson.version}`

const action_url = `https://${namespace}.${hostname}/api/v1/web/${runtimePackage}/<%= actionName %>`
```

Note that <%= actionName %> is passed by the action template. 

I think we should have a user library that would retrieve the URL for the user but that's not available right now.